_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title:  "ETL in Trino speeds with project Tardigrade: now ready for early users"
+author: Andrii Rosa, Brian Zhan, Lukasz Osipiuk, Zebing Lin
+excerpt_separator: <!--more-->
+---
+
+After 6 months of challenging work, we’ve launched Project Tardigrade. It is 
+designed to improve user experience of running resource intensive queries that
+are common in ETL space. It’s available right now! It required some significant
+and fascinating engineering to get us here. Read on to learn how Project 
+Tardigrade works and how to enable it. We think you’ll enjoy hearing about it.
+
+<!--more-->
+
+## What is Project Tardigrade
+
+What we love most about doing ETL with Trino is that you get fast query speeds,
+ and you can iterate fast with intuitive error messages, interactive 
+ experience, and query federation.
+
+One of the big problems that had bugged us for a long time is that doing 
+long-running ETL workload with Trino is hard. You've probably heard of the 
+below issues:
+
+* Reliable landing times: Queries that run for hours can fail and restart 
+  from scratch. This wastes CPU resources and makes it hard for you to meet 
+  your data landing SLA. 
+* Cost-efficient clusters: Trino queries that need terabytes of distributed 
+  memory require extremely large clusters today due to lack of iterative 
+  execution.
+* Concurrency: Multiple independent clients may submit their queries 
+  concurrently. Due to the lack of available resources at a certain moment some
+  of these queries may need to be killed and restarted from zero after a 
+  while. This makes the landing time hardly predictable.
+
+While it is certainly possible to 
+[structure your workload to avoid these problems](https://engineering.salesforce.com/how-to-etl-at-petabyte-scale-with-trino-5fe8ac134e36) 
+it might not be easy to maintain for every company and may not always be
+possible for certain use cases.
+
+Project Tardigrade is designed to provide an "out of the box" solution for the
+aforementioned problems. We’ve created an innovative architecture (you can read
+about it [here](https://docs.google.com/document/d/18QQy8dkZQDNiBPEe3bsyt3RhImCKye4fUJB8bvoeLBI)
+and track ongoing developments 
+[here](https://github.com/trinodb/trino/issues/9101)) that allows us to 
+implement an advanced resource aware scheduling with granular retries. 
+What this means is:
+* When your long-running queries experience a failure, they don’t have to start
+  from scratch
+* When queries require more memory than currently available in the cluster they
+  would still be able to succeed
+* When multiple queries are submitted concurrently they would be able to share
+  resources in a fair way and make steady progress
+
+Trino will do all the hard work behind the scenes of allocating, configuring,
```

Comment:
Trino does all the hard work of allocating, configuring and maintaining query processing behind the scenes.